src/index.html

```diff
@@ -7,8 +7,27 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <div class="wrap">
+      <header class="header">
+        <img class="logo" src="logo.png" alt="logo">
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__link" ><a class="link apple is-active" href="#">Apple</a></li>
```

Comment:
1) apply code styling recommendations for other cases as well
2) change class names
```suggestion
            <li class="nav__item" >
              <a class="nav__link is-active" href="#">
                Apple
              </a>
            </li>
```